File: task-03/sub-task 2/Go/code.go
package main

import (
	"log"
	"os"
)

func main() {
	// Read the file using os.ReadFile
	content, err := os.ReadFile("/home/abinav/git/amfoss-tasks/task-03/sub-task 2/input.txt")
	if err != nil {
		log.Fatal(err)
	}

	// Write the file using os.WriteFile
	err = os.WriteFile("/home/abinav/git/amfoss-tasks/task-03/sub-task 2/Go/output.txt", content, 0644)
	if err != nil {
		log.Fatal(err)
	}
}

File: task-03/sub-task 2/Javascript/code.js
const fs = require('fs');

fs.readFile('/home/abinav/git/amfoss-tasks/task-03/sub-task 2/input.txt', 'utf8', (err, data) => {
    if (err) throw err;
    fs.writeFile('/home/abinav/git/amfoss-tasks/task-03/sub-task 2/Javascript/output.txt', data, (err) => {
        if (err) throw err;
    });
});
